### ISSUE TITLE
Lint to detect npm dependencies without versions

### ISSUE THREAD

**Issue by sigmaSd** [denoland/deno_lint#1227]
code

```ts
import {} from "npm:chalk"
```

should warn `Implicitly using latest version of chalk ..`

The deno module is equivalent is implemented as  `x-deno-warning` header provided by dotland , so we can't expect npm to provide something similar, so maybe a lint can work instead

**Comment by lancy69** [comment]
Hi! I think #1362 already fixed this. This issue appears to be stale and should likely be closed.

**Comment by sigmaSd** [comment]
Hi, indeed its fixed thanks
